server/routers/sessions.js

```diff
@@ -1,6 +1,6 @@
 const router = require('express').Router();
 const bcrypt = require('bcrypt');
-const jwt = require('jwt');
+const jwt = require('jsonwebtoken');
```

Comment:
Is this right? Because in the `authentication-middleware` I also used `jwt` and if that runs this should as well